jbs/src/main/java/org/openjdk/skara/jbs/JdkVersion.java

```diff
@@ -39,7 +39,7 @@
     private final static Pattern fxVersionPattern = Pattern.compile("(openjfx[1-9][0-9]?)(u([0-9]{1,3}))?$");
 
     // Match a version string symbolizing some future, but yet undefined, update of a major version
-    private final static Pattern futureUpdatePattern = Pattern.compile("([1-9][0-9]*u)(-([a-z0-9]+))?$");
+    private final static Pattern futureUpdatePattern = Pattern.compile("((openjdk)?[1-9][0-9]*u)(-([a-z0-9]+))?$");
```

Comment:
This seems like it has some overlap with `ojVersionPattern`. It might be OK, though.

Reply:
Right, there is certainly overlap between some of these expressions. In the end I picked this one to extend as I thought the most defining feature was the 'u' without any number after it.